JS/Lesson17/onder/script.js

```diff
@@ -134,19 +134,54 @@ console.log('----------------------------');
 // Input: ({name: "John", age: 30}, "city", "New York")
 // Output: {name: "John", age: 30, city: "New York"}
 
+//I'm really not sure about 10-11-12-13 questions.
+const object = { name: "John", age: 30 };
+const key = "city";
+const value = "New York";
+
+const result = addObjectKeyValue(object, key, value);
+console.log(result); 
+   
+
+
 console.log('----------------------------');
 // 11. Write a function that takes an object and a key, and removes the key-value pair from the object.
 // Input: ({name: "John", age: 30, city: "New York"}, "city")
 // Output: {name: "John", age: 30}
 
+function removeObjectKey(obj, key) {
+    delete obj[key];
+    return obj;
+  }
+  const object = { name: "John", age: 30, city: "New York" };
+  const key = "city";
+  
+  const result = removeObjectKey(object, key);
+  console.log(result);
+  
 console.log('----------------------------');
+
 // 12. Write a function that takes two arrays of equal length and returns an object with the first array as keys and the second array as values.
 // Input: (["name", "age", "city"], ["John", 30, "New York"])
 // Output: {name: "John", age: 30, city: "New York"}
 
+const keys = ["name", "age", "city"];
+const values = ["John", 30, "New York"];
+
+const object = arraysToObject(keys, values);
```

Comment:
Check this suggestion: https://github.com/frontend-from-0/240123-exercises/pull/50#discussion_r1307932841